src/accelerate/accelerator.py

```diff
@@ -272,7 +272,7 @@ def __init__(
             self.native_amp = is_torch_version(">=", "1.6")
             if not self.native_amp:
                 raise ValueError(err.format(mode="fp16", requirement="PyTorch >= 1.6"))
-            if not torch.cuda.is_available():
+            if not torch.cuda.is_available() and os.environ.get("USE_MPS_DEVICE", "false") == "false":
```

Comment:
Can we do something similar to this [here](https://github.com/huggingface/accelerate/blob/main/src/accelerate/test_utils/testing.py#L94-L99) so that cuda tests can also be ran? As so far tests are only running on the CPU here 😄

Reply:
Hello, I didn't understand. Let's take this offline to better understand.

Reply:
Resolving based on Sylvains note about the tests